lib/registry/security/security.go

```diff
@@ -120,6 +126,34 @@ func (c Config) getCredentials(helper, addr string) (types.AuthConfig, error) {
 }
 
 func (c Config) getCredentialFromHelper(helper, addr string) (types.AuthConfig, error) {
+	if strings.HasPrefix(helper, "ecr") {
```

Comment:
maybe match the full name `ecr-login` would be better here?

Reply:
Switched to switch case